index.js

```diff
@@ -21,7 +21,7 @@ const customConfig = {};
 
 const getConfig = function getConfig() {
   const args = [].slice.call(arguments);
-  return Object.assign.apply(null, [{}, defaults, customConfig].concat(args));
+  return _.merge.apply(_, [{}, defaults, customConfig].concat(args));
```

Comment:
ah yes, I didn't look at the removed code! :D wouldn't _.mergeDeep be more robust if combining things that could be multiple levels deep?